odoo/tests/addons/test_translation_import/i18n/es.po

```diff
@@ -0,0 +1,17 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* test_translation_import
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0alpha1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-07-05 11:04+0000\n"
+"PO-Revision-Date: 2017-07-05 11:04+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
```

Comment:
You should be able to drop this file and use StringIO buffers for the test instead.